src/RA_AchievementOverlay.cpp

```diff
@@ -1583,41 +1584,38 @@ void AchievementOverlay::InstallNewsArticlesFromFile()
 {
     m_LatestNews.clear();
 
-    FILE* pf = nullptr;
-    fopen_s(&pf, RA_NEWS_FILENAME, "rb");
-    if (pf != nullptr)
-    {
-        Document doc;
-        doc.ParseStream(FileStream(pf));
-
-        if (doc.HasMember("Success") && doc["Success"].GetBool())
-        {
-            const Value& News = doc["News"];
-            for (SizeType i = 0; i < News.Size(); ++i)
-            {
-                const Value& NextNewsArticle = News[i];
-
-                NewsItem nNewsItem;
-                nNewsItem.m_nID = NextNewsArticle["ID"].GetUint();
-                nNewsItem.m_sTitle = NextNewsArticle["Title"].GetString();
-                nNewsItem.m_sPayload = NextNewsArticle["Payload"].GetString();
-                nNewsItem.m_sAuthor = NextNewsArticle["Author"].GetString();
-                nNewsItem.m_sLink = NextNewsArticle["Link"].GetString();
-                nNewsItem.m_sImage = NextNewsArticle["Image"].GetString();
-                nNewsItem.m_nPostedAt = NextNewsArticle["TimePosted"].GetUint();
 
-                tm destTime;
-                localtime_s(&destTime, &nNewsItem.m_nPostedAt);
+    std::ifstream ifile{ RA_NEWS_FILENAME, std::ios::binary };
```

Comment:
lots of these files from the macros look like json files when I open them, is there a reason why they use the .txt extension?

Reply:
Probably